draft-scenarios/clash/dragon_valley.tex

```diff
@@ -0,0 +1,129 @@
+  \item 2 VP for each enemy Main Town captured - \textit{once per captured Faction.}
+  \item 1 VP for every controlled Mine or Settlement.
+  \item 1 VP for every Level of Experience of the Main Hero.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\begin{itemize}
+  \item Start of \textbf{\nth{1} and \nth{9} Rounds:} All Heroes gain +1 Movement.
+  \item Start of \textbf{\nth{4} and \nth{8} Rounds:} Remove black Cubes from all locations that give resources, morale or Cards.
+  \item End of \textbf{\nth{10} Round:} Player(s) with lowest XP roll 2 Resource dice.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+  \item Remove “Earthquake”, “Remove Obstacles”, “View Earth” and “Visions" from the spell Deck.
+  \item Remove “Unexpected Reinforcements” from the Astrologers Proclaim Deck.
+  \item Make one Deck of minor, one of major and one of relic artifacts.
+  \item Make one Deck of basic spells and one of expert spells.
+  \item On Starting and Far Tiles players can only get basic spells and minor artifacts.
+  \item Players may not use Diplomacy to skip lvl VII Combat.
+  \item If your Hero is lvl VII, treat lvl VI neutrals as lvl VII and gain 10 \svg{gold}
+  \item Sanctuary effect: +1 Morale and draw 1 Card. Visitable once per Faction.
+  \item Obelisk effect: Roll one Resource die and draw one Card. \textit{May be visited once per Faction.}
+  \item When fighting a neutral Combat, a player may add +1 difficulty for an extra 5 \svg{gold} + Search (2) the Artifact or the Spell Deck
+  \item Level VII settlements are treated as two settlements for rewards AND roll one Resource or Treasure (reroll any XP) die.
+  \item If a player defeats neutral Units on a Grail hex, he immediately gets: 10 \svg{gold}, 4 \svg{building_materials} and 2 \svg{valuables}. (alternative: Throw 2 treasure (reroll any XP ) and 2 Resource dice)  \textit{Can be conquered only once.}
+  \item Dragon Utopia: Reshuffle the \svg{gold} and \svg{azure} Decks, then draw \svg{azure} until you get 3 Dragons and choose 2 AND draw from \svg{gold} until you get one Dragon. 
+  \item When a player conquers Dragon Utopia, he may recruit one of the neutral Dragons killed for half the \svg{gold} (rounded down) + \svg{valuables} OR gain 15 \svg{gold} AND Search (3) any artifact Deck. \textit{Can be conquered only once.}
+\end{itemize}
+
+When a Dragon Utopia is conquered, roll an attack die:
+\begin{itemize}
+  \item -1: The defeated Dragons’ spirits curse the land! All players lose 1 MP for the next Round.
+  \item  0: Dragons revenge: All players must sacrifice one Unit before the end of Round, that Unit's health is reduced to 0. 
+  \item +1: Scattered Treasure: All players may immediately lose 1 MP to Search (2) the artifact Deck.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Suggested houserules}}
+\begin{itemize}
+  \item Trading Posts - Trade resources and remove Cards, OR buy war machines.
```

Comment:
```suggestion
  \item Trading Posts - on a single visit, trade Resources and remove Cards, OR buy a War Machines.
```
I take it that you can trade resources and remove as many cards as you want, all on a single MP?